PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-list/pokemon-list.component.html

```diff
@@ -1,973 +1,997 @@
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">ATK</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Atk, -1)" 
+                    (click)="sort(column.Atk, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-defense">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Def, 1)" 
+                    (click)="sort(column.Def, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">DEF</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Def, -1)" 
+                    (click)="sort(column.Def, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-special-attack">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spa, 1)" 
+                    (click)="sort(column.Spa, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">SPA</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spa, -1)" 
+                    (click)="sort(column.Spa, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-special-defense">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spd, 1)" 
+                    (click)="sort(column.Spd, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">SPD</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spd, -1)" 
+                    (click)="sort(column.Spd, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-speed">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spe, 1)" 
+                    (click)="sort(column.Spe, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="table-header-label">SPE</div>
+                <div 
+                    [ngClass]="getSortButtonClass(column.Spe, -1)" 
+                    (click)="sort(column.Spe, -1)"
+                    class="table-header-sort">⮟</div>
+            </th>
+            <th class="col-stat stat-bulk">
+                <div 
+                    [ngClass]="getSortButtonClass(column.Bulk, 1)" 
+                    (click)="sort(column.Bulk, 1)"
+                    class="table-header-sort">⮝</div>
+                <div class="has-tooltip">
+                    <div class="table-header-label">Bulk</div>
+                    <span class="tooltip">Bulk can be calculated by adding up the Healthpoints(HP), Defense(DEF) and Special Defense(SPD) stats. It can be used to search for Pokémon which are hard to defeat.</span>

```

Comment:
Tooltip text should be much shorter, i.e. "Bulk = HP + Def. + Sp.D.". Only a quick explanation how the value is calculated should be sufficient.

Reply:
is fixed now